### ISSUE TITLE
Fonts/text showing as boxes

### ISSUE THREAD

**Issue by Enteneller2017** [nsg/snap-springlobby#3]
![solus_springlobby-nsg_fonts](https://user-images.githubusercontent.com/29702072/38576349-2cb12104-3cfe-11e8-86a8-1d91ac3f443b.jpg)
OS: Solus Linux

**Comment by nsg** [comment]
Yeah, this is annoying :) I have got the same report from a Debian user. The snap uses the host operating system fonts, and it look like it requires something only installed on Ubuntu systems (possible also Fedora). I had no idea when I made the snap that this was the case :\

Can you take a look what `fc-list` returns on your system? Do you have dejavu installed?

**Comment by Enteneller2017** [comment]
Dejavu-fonts-ttf is installed. fc-list returns an inordinate list of .ttf files.

**Comment by nsg** [comment]
I see,

I think I will install Solus in a VM to debug it my self. At the moment I'm a little blocked by the fact that something has changes in either snapcraft, snapd or some dependency because if I build the application again, I get a crash. Need to figure this one out before I release a new version with the font fix.

**Comment by nsg** [comment]
I'm merging a fix now (see: #5) to master, that will be available in the edge channel in about an hour.

`snap refresh --edge springlobby-nsg` or `snap install --edge springlobby-nsg`

Did that fix it for you? It looks good in my test VM.

**Comment by Enteneller2017** [comment]
The fonts work now, thanks and good job. 

When I move to the Singleplayer tab, I can't choose a map nor a mod. Manually locating the spring binary and unitsync files didn't work, I've joined an online game and had the client download a working spring binary 104.0. It is now displayed in Singleplayer, but the "Game" and "Map" dropdowns are still empty. According to the preferences the binary used is in snap/springlobby-nsg/common/.spring/engine/104.0/spring. I've copied the folders "games" and "maps" to numerous subfolders to have the client recognise them, no luck. Any ideas?
